keyboards/planck/keymaps/vuldriel/config.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Do I use my actual name? Sorry I'm really new to this.

Reply:
Doesn't need an actual name, you can use your github user name.  Or you can put `QMK` there, instead of a name.

Reply:
I pushed a new `keymap.c`.